propertyestimator/utils/utils.py

```diff
@@ -19,7 +19,7 @@ def find_types_with_decorator(class_type, decorator_type):
     ----------
     class_type: class
         The class to pull attributes from.
-    decorator_type: str
+    decorator_type: type
         The type of decorator to search for.
 
```

Comment:
In the new code, is there a difference between `type`s and `class`es? [This SO post](https://stackoverflow.com/a/4479888) seems to say that they're the same thing in python 3.

Reply:
I _think_ the two are the same now although I'm never sure which to use for docstrings - I have swapped the old `class` to `type` in b8a79e30730682cb2478e4d77ac2ffd4b44b4b9e to at least be consistent here!